### PR TITLE
Setup sonar backend

### PR DESCRIPTION
**Pull request by AndriiNasadiuk**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npm run pre-commit
+git config advice.ignoredHook false
```
Comment (online-marathon) [inline .husky/pre-commit]: We realy need this line? @Mygal-Alex , @LeonidShamarin , @ArtemHolikov , @marianamartsy , please test how it work for you.

File: .husky/pre-push [hunk 2]
```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npm run test
+git config advice.ignoredHook false
```
Comment (online-marathon) [inline .husky/pre-push]: We realy need this line? @Mygal-Alex , @LeonidShamarin , @ArtemHolikov , @marianamartsy , please test how it work for you.

**Review by online-marathon — CHANGES_REQUESTED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (marianamartsy) [reply .husky/pre-commit]: no, we don't need this line, I guess

Comment (AndriiNasadiuk) [reply .husky/pre-commit]: Yes, i deleted that line

**Pull request merged**